pkg/network/tags_nolinux.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !linux
+// +build !linux
+
+package network
+
+// ConnTag constant must be the same for all platform
+const (
+	GnuTLS  ConnTag = 1
```

Comment:
What do you think about having these constants only be defined in a test file? We don't actually need the constants in non-test code.

Reply:
https://github.com/DataDog/datadog-agent/commit/39357f6ec15fe0e98f8f2326d80de1335ceaea88